lib/tests/fixtures/facts/external/windows/powershell/json.ps1

```diff
@@ -0,0 +1 @@
+Write-Output (@{ 'ps1_json_fact1'='value1'; 'PS1_JSON_fact2' = 'value2' } | ConvertTo-JSON -Depth 1 -Compress)
```

Comment:
Just an FYI (that we might want to add to docs somewhere) - `ConvertTo-JSON` is only available in PowerShell 3 and up... and there are servers in the wild still running PowerShell 2.

Reply:
I would suggest making the test a bit more comprehensive, to make sure types are mapped across properly from PowerShell -> JSON:

```powershell
@{ 
  'ps1_json_fact1'='value1'
  'PS1_JSON_fact2' = 2
  'PS1_JSON_fact3' = $true
  'PS1_JSON_fact4' = @('uno', 'dos')
  'PS1_JSON_fact5' = $Null
  'PS1_JSON_fact6' = @{ 'key' = 'value' }
 } | ConvertTo-JSON -Depth 1 -Compress
```

This should produce a JSON string like:

```json
{"PS1_JSON_fact6":{"key":"value"},"PS1_JSON_fact3":true,"PS1_JSON_fact4":["uno","dos"],"ps1_json_fact1":"value1","PS1_JSON_fact2":2,"PS1_JSON_fact5":null}
```

We tested something similar in the JSON -> PS mapping that's done as part of PE tasks.  I have a more comprehensive list of common types available in https://gist.github.com/Iristyle/3b37ad44c93e3e00502d799344294d93 that we vetted.